README.md

```diff
@@ -113,6 +113,9 @@ Import Mandrill blacklist that you get from [here](https://mandrill.zendesk.com/
 
 If the list was successfully imported the CLI will return `OK`.
 
+### Import SendGrid Suppressions
+
+Follow [SendGrid Suppressions import guide](Usage_Sendgrid_Suppressions.md).
```

Comment:
Include a tl;dr for experienced folk? Something like `tl;dr - the CLI can import a CSV with the following fields: ...`